### DISCUSSION TITLE
How to implement multiple endpoints?

### DISCUSSION THREAD

**Discussion by niti-alzhemist** [zino-app/graphql-flutter#1218] [Q&A] [answered]
In a micro-service, we have endpoint like 
`https:www.example.com/service-a/graphql`
`https:www.example.com/service-b/graphql`
`https:www.example.com/service-c/graphql`

How can I switch between them? 

Thanks

**Comment by budde377** [comment, marked as answer]
You need to use three different clients.

**Reply by niti-alzhemist** [reply]
Thank you !
